backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -17,6 +24,8 @@ body {
   @media print { display: none }
 }
 
+.admin-nav-extended { padding-left: 0; }
+
```

Comment:
Here you could add all the CSS magic for showing and hiding the nav bar. For instance:

```scss
.admin-nav {
  transform: translate3d(0, 0, 0);
  transition: transform 350ms ease-in-out;
}

#admin-nav-toggle . fa-chevron-right {
  transform: rotate(0);
  transition: transform 350ms linear;
}

.admin-nav-extended { 
  padding-left: 0; 

  .admin-nav { 
    transform: translate3d(-100%, 0, 0);
  }

  #admin-nav-toggle .fa-chevron-right {
    transform: rotate(180deg);
  }
}
```